Relation extraction and knowledge graph creation

Dylan-Gallagher opened the pull request:
Implemented relation extraction on the AI Act using the REBEL model. Used the relations generated from it to create a knowledge graph output.

rbarrtcd reviewed (changes requested):
2 notes:
1: Add torch to the requirements.txt file

2: The net.show line in rebel_re_model.py at line 105 has been causing errors for me.
This is because it only works if the user's default browser is currently already open (from my research and testing).

I reccomend using
net.save_graph("knowledge_graph/knowledge_graph.html")
instead
and/or putting the net.show in a try except statment.

Dylan-Gallagher commented:
Implemented requested changes.
- Added link instructions to install PyTorch with GPU support.
- Added torch to requirements.txt
- Changed code to `net.save_graph("knowledge_graph/knowledge_graph.html")`
